packages/common/src/context/comments/commentsContext.tsx

```diff
@@ -223,7 +223,7 @@ export function CommentSectionProvider<NavigationProp>(
   )
 
   const commentSectionLoading =
-    (status === 'loading' || isFetching) && !isLoadingMorePages
+    (status === 'pending' || isFetching) && !isLoadingMorePages
```

Comment:
is this a change in react-query? is it a different state or just the new name for `loading`?